base/exports.jl

```diff
@@ -1180,8 +1180,10 @@ export
 # multiprocessing
     addprocs,
     asyncmap,
+    CachingPool,
     ClusterManager,
     default_worker_pool,
+    define,
```

Comment:
if we can't say "this is what this verb should mean" then maybe it's not a name we should be exporting as a generic function yet